templates/microcloud/roadshow.html

```diff
@@ -0,0 +1,268 @@
+{% extends 'base_index.html' %}
+
+{% block title %}Canonical MicroCloud Roadshow{% endblock %}
+
+{% block description %}Canonical MicroCloud Roadshow is the right place to talk about how this lightweight open source cloud platform alleviates the usual burdens of cloud deployment and management. Meet us in Europe this winter and let's talk about MicroCloud.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1f5r_jSdTKPTTF0rj-82MC4kAODwjGlts4HLehAk_hnA/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6 col-medium-3">
+      <div class="p-block">
+        <h1>
+          The Canonical MicroCloud Roadshow
+        </h1>
+      </div>
+    </div>
+    <div class="col-6 col-medium-3">
+      <p>Deploying your own cloud typically involves significant investments &ndash; in terms of
+        costs, time and skills. MicroClouds help minimise these investments by providing you with a cost-effective
+        open source cloud that deploys in minutes and is easy to operate and maintain. Join us on the roadshow to
+        see it in action and discover how you can benefit by using MicroCoud.</p>
+      <p class="u-no-margin--bottom">
+        <a href="#find-us" class="p-button--positive u-no-margin--bottom">Check where to find us</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6 col-medium-3">
+      <div class="p-block">
+        <h2>What is a MicroCloud?</h2>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6 col-start-large-7">
+    <iframe width="603" height="339" src="https://www.youtube-nocookie.com/embed/FnzUcGg60Qk" frameborder="0"
+            title="What is a MicroCloud?"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            loading="lazy" allowfullscreen="" class="responsiveIframe u-no-margin--left"></iframe>
```

Comment:
I don't love this kind of indentation (either we indent all attributes or put everything in one line, I always use editor to wrap code if necessary), but it's not a blocker.

Reply:
I don't either tbh, updated